src/Share.jsx

```diff
@@ -47,8 +49,8 @@ export default function Share({installPrefix, appPrefix, pathPrefix}) {
         return
       }
       if (modelPath === null ||
-        (modelPath.filepath && modelPath.filepath !== mp.filepath) ||
-        (modelPath.gitpath && modelPath.gitpath !== mp.gitpath)) {
+          (modelPath.filepath && modelPath.filepath !== mp.filepath) ||
```

Comment:
not sure whether this tab space is correct.

Reply:
It is.  Doesn't look right here but is fine in my client.